webstruct/tests/test_date_features.py

```diff
@@ -0,0 +1,71 @@
+from webstruct.features import (looks_like_ordinal,
+                                looks_like_date_pattern,
+                                number_looks_like_day,
+                                number_looks_like_month)
+
+
+class HtmlToken():
+    def __init__(self, token):
+        self.token = token
+
+
+def test_looks_like_ordinal():
+
+    def assert_looks_like_ordinal(token, expected):
+        assert looks_like_ordinal(token) == expected
```

Comment:
If you replace it with 

```py
assert looks_like_ordinal(HtmlToken(text)) == {'looks_like_ordinal': expected}
```

test code will be smaller and more DRY, it'd be easier to add more tests.